moz_security/tests/hindsight/run/input/generate_sshd.lua

```diff
@@ -30,7 +30,7 @@ function process_message()
         msg.Uuid = v[1]
         msg.Fields.remote_user = v[2]
         msg.Fields.remote_addr = v[3]
-        msg = ga.geo_append(msg)
+        geo.add_geoip(msg, "remote_addr")
```

Comment:
I'm guessing even though it's like this in the test, in the implementation we'd have the fields in the sandbox configuration (e.g., `fields = { "remote_addr", "etc" }`)

Reply:
No, this cannot be cfg driven (at least not from the geoip_heka cfg) as the field can be variable.  Some logs my have source_ip others remote_addr all handled by the same decoder. See the example in https://github.com/mozilla-services/lua_sandbox_extensions/issues/220